### PR TITLE
add docs for podman on windows

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: content/en/references/podman.md [hunk 1]
```diff
@@ -70,3 +70,46 @@ If you have problems with [subuid and subgid](https://wiki.archlinux.org/title/P
 ```sh
 DEBUG=1 DOCKER_CMD="podman --storage-opt overlay.ignore_chown_errors=true" DOCKER_SOCK=$XDG_RUNTIME_DIR/podman/podman.sock DOCKER_HOST=unix://$XDG_RUNTIME_DIR/podman/podman.sock localstack start
 ```
+### Podman on Windows
+
+You can run Podman on Windows using WSL2. In the guide, we use a Docker Compose setup to run LocalStack.
```
Comment (MarcelStranak) [inline content/en/references/podman.md:75]:
```suggestion
You can run Podman on Windows using [WSLv2](https://learn.microsoft.com/en-us/windows/wsl/about#what-is-wsl-2). In the guide, we use a Docker Compose setup to run LocalStack.
```

File: content/en/references/podman.md [hunk 2]
```diff
@@ -70,3 +70,46 @@ If you have problems with [subuid and subgid](https://wiki.archlinux.org/title/P
...
+$ podman machine start
+{{< / command >}}
+
+At this stage, Podman operates in rootless mode, where exposing port 443 on Windows is not possible. To enable this, switch Podman to rootful mode using the following command:
+
+{{< command >}}
+podman machine set --rootful
+{{< / command >}}
+
+For the Docker Compose setup, use the following configuration. In rootless mode, exclude the HTTPS gateway port by commenting it out, since it cannot bind to privileged ports under 1024:
```
Comment (MarcelStranak) [inline content/en/references/podman.md:90]:
```suggestion
For the Docker Compose setup, use the following configuration. When running in rootless mode, ensure to comment out the HTTPS gateway port, as it is unable to bind to privileged ports below 1024.
```

**Review by MarcelStranak — COMMENTED** [review]
LGTM. Thanks for adding it.

**Review by dfangl — COMMENTED** [review]
The same instructions (without the :443 notice, that one works on rootless if bound to `0.0.0.0`, which is why it is like that in the docker compose file) are also applicable to mac. Could we adapt it, so we have instructions for it as well?

**Pull request merged**